jobsdb/readonly_jobsdb.go

```diff
@@ -219,7 +219,7 @@ func (jd *ReadonlyHandleT) getUnprocessedJobsDSCount(ctx context.Context, ds dat
 
 	var sqlStatement string
 
-	sqlStatement = fmt.Sprintf(`LOCK TABLE %s IN ACCESS SHARE MODE;`, ds.JobStatusTable)
+	sqlStatement = fmt.Sprintf(`LOCK TABLE %q IN ACCESS SHARE MODE;`, ds.JobStatusTable)
```

Comment:
Queries in other methods should need adaptations too, e.g. `getUnprocessedJobsDSCount` & `getProcessedJobsDSCount`

Reply:
Changed them

Reply:
Nonetheless, there is an existing test that [is failing](https://github.com/rudderlabs/rudder-server/actions/runs/3676813996/jobs/6217937419)

Reply:
Will close this PR as it doesn't seem very straightforward to incorporate negative index support for these queries.Let's try and get rid of them by next one